scripts/generators/create-component/index.ts

```diff
@@ -6,10 +6,11 @@ import * as fs from 'fs-extra';
 import * as os from 'os';
 import * as path from 'path';
 import { execSync } from 'child_process';
-import { findGitRoot, getAllPackageInfo, isConvergedPackage } from '../monorepo/index';
```

Comment:
fixed type errors and normalized name to follow our nx generarators -> `index.ts` as root